src/components/Blocks/ChatBubble/ChatBubble.tsx

```diff
@@ -1,43 +1,31 @@
-import Image from 'next/image';
+import { ChatBubbleLeft } from '@/components/Icons/ChatBubbleLeft/ChatBubbleLeft';
+import { ChatBubbleRight } from '@/components/Icons/ChatBubbleRight/ChatBubbleRight';
```

Comment:
Why are we rendering svgs as components?

Reply:
That's a pattern I've seen on the project I'm working on at work. I think it makes them much cleaner and more reusable. What do you think, good or bad?

Reply:
Maybe IDK enough about SVG's, I was hoping there was something more here. I did a quick search, and apparently they're meant to be HTML and not .svg. So coloring via CSS is a bit more complicated. Since this is just code, is it possible to make it 1 SVG component, and since we're rendering it, make it Left or Right based on component properties?

Reply:
Yeah, definitely doable. I like it. Implemented!